documentation/book/assembly-storage.adoc

```diff
@@ -25,6 +25,13 @@ When configuring a `Kafka` resource, you can specify the type of storage used by
 
 The storage type is configured in the `type` field.
 
+.Additional resources
+
+* For more information about ephemeral storage, see xref:type-EphemeralStorage-reference[ephemeral storage schema reference].
+* For more information about persistent storage, see xref:type-PersistentClaimStorage-reference[persistent storage schema reference].
+* For more information about JBOD storage, see xref:type-JbodStorage-reference[JBOD schema reference].
+* For more information about the schema for `Kafka`, see xref:type-Kafka-reference[`Kafka` schema reference].
+
 WARNING: The storage type cannot be changed after a Kafka cluster is deployed.
```

Comment:
If the users miss this warning because it is only after the links they might be in for a unpleasant surprise. Should we move this before the links?

Reply:
True. Moved above the Additional resources. 
![image](https://user-images.githubusercontent.com/47596553/61369619-654bdd00-a888-11e9-8a16-6420447412f1.png)